src/diagonal.works/b6/api/functions/change.go

```diff
@@ -155,11 +157,14 @@ func addCollection(c *api.Context, id b6.CollectionID, tags b6.Collection[any, b
 	return &add, nil
 }
 
+// TO(andrew): doubting this now a little bit - maybe we should do basic add feature ? (just like empty w id)
```

Comment:
I'd leave it like this personally. I think it's good to have the expression tag hidden inside the function for the moment. But once we're done with this, we can rationalise, and ensure there is a, eg, newXFeature for each value type X.

Reply:
sg